service/src/intTest/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentComponentTest.java

```diff
@@ -38,6 +39,7 @@
     private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
     private static final String DOCUMENT_NAME = EhrStatusConstants.DOCUMENT_ID + ".json";
     private static final String INVALID_DOCUMENT_ID = "invalid-id";
+    private static final String MHS_FILE_NAME_TEMPLATE = "%s_doc_translated.json";
```

Comment:
unused, should it be?

Reply:
We are uploading one document with "{documentId}.json" format.